src/overlays/actors/ovl_En_Torch2/z_en_torch2.c

```diff
@@ -10,12 +10,51 @@
 
 #define THIS ((Player*)thisx)
 
+typedef enum {
+    /* 0 */ ENTORCH2_WAIT,
+    /* 1 */ ENTORCH2_ATTACK,
+    /* 2 */ ENTORCH2_DEATH,
+    /* 3 */ ENTORCH2_DAMAGE
+} EnTorch2ActionStates;
+
+typedef enum {
+    /*  0 */ FORWARD_SLASH_1H,
+    /*  1 */ FORWARD_SLASH_2H,
+    /*  2 */ FORWARD_COMBO_1H,
+    /*  3 */ FORWARD_COMBO_2H,
+    /*  4 */ RIGHT_SLASH_1H,
+    /*  5 */ RIGHT_SLASH_2H,
+    /*  6 */ RIGHT_COMBO_1H,
+    /*  7 */ RIGHT_COMBO_2H,
+    /*  8 */ LEFT_SLASH_1H,
+    /*  9 */ LEFT_SLASH_2H,
+    /* 10 */ LEFT_COMBO_1H,
+    /* 11 */ LEFT_COMBO_2H,
+    /* 12 */ STAB_1H,
+    /* 13 */ STAB_2H,
+    /* 14 */ STAB_COMBO_1H,
+    /* 15 */ STAB_COMBO_2H,
+    /* 16 */ FLIPSLASH_START,
+    /* 17 */ JUMPSLASH_START,
+    /* 18 */ FLIPSLASH_FINISH,
+    /* 19 */ JUMPSLASH_FINISH,
+    /* 20 */ BACKSLASH_RIGHT,
+    /* 21 */ BACKSLASH_LEFT,
+    /* 22 */ HAMMER_FORWARD,
+    /* 23 */ HAMMER_SIDE,
+    /* 24 */ SPIN_ATTACK_1H,
+    /* 25 */ SPIN_ATTACK_2H,
+    /* 26 */ BIG_SPIN_1H,
+    /* 27 */ BIG_SPIN_2H
+} PlayerSwordAnimation;    
```

Comment:
def would be pretty weird to have this enum for dark link and not use it with player as well.
@Roman971 not sure if you want to wait for a player doc pass, but yeah just my opinion

Reply:
It's kind of here as a plant for Player to use it. And there's little reason not to (it's definitely correct) aside from the fact that implementing it into z_player would likely take a bit of time.

Reply:
It's definitely going to be weird until we have unified everything, but since it's already there and used in dark link, I think it's fine to leave at that for the time being.